app/main.py

```diff
@@ -1,3 +1,40 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.car import Car
+from app.shop import Shop
+from app.customer import Customer
+import json
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as json_file:
+        data_file = json.load(json_file)
+        fuel_price = data_file["FUEL_PRICE"]
+        shop_dict = {}
+    for item in data_file["customers"]:
+        customer = Customer(item["name"],
```

Comment:
After opening the parentheses, write all the arguments on a new line, including the first one